client/src/components/Events/Event.js

```diff
@@ -1,25 +1,30 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../../styles/events.css';
 
 // Event appears on EventListPage
 const Event = ({ id, date, time, location, eventTitle }) => {
-	const eventStyle = {
-		border: '1px solid black',
-		padding: '15px',
-		marginTop: '8px',
-		marginLeft: '4px',
-		marginRight: '4px',
-	}
+  //const eventStyle = {
+  //border: '1px solid black',
+  //padding: '15px',
+  //marginTop: '8px',
+  //marginLeft: '4px',
+  //marginRight: '4px',
+  //}
 
-	return (
-		<div className="pancake-child" style={eventStyle}>
-			<h4>
-				<Link to={`/events/${id}`}>{eventTitle}</Link>
-			</h4>
-			<p>Location: {location}</p>
-			<p>Date: {date}</p>
-			<p>Time: {time}</p>
-		</div>
-	)
-}
-export default Event
+  return (
+    <div className="event-container">
+      <div className="pancake-child">
```

Comment:
Why create another div? Why not just apply multiple classes or even consolidate them? Actual questions. I'm not a frontend engineer.

Reply:
depends on how you want to style the parent and child div.

Reply:
There are multiple files with the className="pancake-child". I re-wrapped it in a new div to apply some css only to events